packages/react/src/runtime/is-node.tsx

```diff
@@ -11,6 +11,5 @@
  * ```
  */
 export const isNodeEnvironment = (): boolean => {
-  // https://nodejs.org/api/process.html#process_process_release
-  return typeof process !== 'undefined' && process?.release?.name === 'node';
+  return typeof document === 'undefined';
```

Comment:
Is there a difference between checking for window vs. document?

Reply:
This was part of the spec in the PIR process but looking at the implementation of our SSR babel plugin, `document` and `window` are interchangeable. Let's go with `window` to be more friendly with bundlers by default.

Reply:
Ah, I missed this quick question - we chose `document` here because Emotion uses it for similar checks as well.

Reply:
(`window` is probably just fine too. That's just the only particular reason we chose `document` over `window`)